homeassistant/components/zha/core/channels/general.py

```diff
@@ -170,6 +172,8 @@ class Commissioning(ZigbeeChannel):
 class DeviceTemperature(ZigbeeChannel):
     """Device Temperature channel."""
 
+    REPORT_CONFIG = [{"attr": "current_temperature", "config": (REPORT_CONFIG_MIN_INT, REPORT_CONFIG_MAX_INT, 50)}]
```

Comment:
I'm not sure why this had to be done, it was working without as well. I've done it only because some tests were failing.

Reply:
Has to be done for zcl compliant devices. Xiaomi sends updates regardless.